rpc/gnmi/mixed-schema.md

```diff
@@ -108,50 +109,187 @@ if all transactions succeed should the `SetResponse` indicate success. If any
 of the transactions fail, the contents of all origins MUST be rolled back, and
 an error status returned upon responding to the `Set` RPC.
 
-## Example
+## Special considerations for a setRequest with openconfig and cli origins
+
+### Use cases
+The operational use cases for a setRequest `replace` operation containing both OC and CLI origins include:
+
+1. The ability to support incremental transition from CLI based configuration to OpenConfig configuration.
+2. A device may implement configuration items which are only defined in CLI while other configuration items are supported with OpenConfig.
+3. The desire to use a full configuration replacement strategy.
+
+To support these use cases a special meaning to a setRequest with replace origin `cli` and replace origin `openconfig` is defined.
+
+### SetRequest behavior
+
+A `SetRequest` message using the `replace` operation for both `cli` and `openconfig` origins SHOULD be performed as follows:
 
-### Example: OpenConfig and CLI Data
+1. `origin` `cli` SHOULD appear first.  
+2. The entire contents of the `ascii` encoded `cli` data SHOULD appear in a single update.
+3. The `replace` operation for origin `cli` SHOULD be interpeted as a full configuration replacement.  In other words, the target device SHOULD erase the `cli` configuration and replace it with the contents of the `cli` update message.
```

Comment:
What about when multiple schema sets are also represented as CLI syntax?  We seem to keep mixing the schema sets with the presentation layer here

Reply:
yes, I understand the use case here with the goal of performing a full-config-replace and providing a CLI blob that is guaranteed to have 100% config coverage.

But this indeed confronts the requirement to keep changes scoped to a certain origin only:

<img width="1058" alt="image" src="https://user-images.githubusercontent.com/5679861/215597778-514f166f-959b-42f1-99af-f610e3f84c02.png">

I think a form of a deviation statement needs to be given.

Reply:
Agreed, we will introduce a new operation to clarify this.